core/controller.go

```diff
@@ -216,13 +230,34 @@ func (c *Controller) ListStreamHandler(w http.ResponseWriter, r *http.Request, _
 		return
 	}
 	dto := []*SummariseDTO{}
+
+	// active streams
 	for key, stream := range c.streams {
+		aliasname := ""
+		for name, id := range c.alias {
+			if id == stream.ID {
+				aliasname = name
+				key = name
+			}
+		}
 		dto = append(dto, &SummariseDTO{
 			URI:     fmt.Sprintf("/stream/%s/index.m3u8", key),
 			Running: stream.Streak.IsActive(),
 			ID:      stream.ID,
+			Alias:    aliasname,
 		})
 	}
+
+	// preload streams
+	for name, _ := range c.preload {
+		dto = append(dto, &SummariseDTO{
+			URI:     fmt.Sprintf("/stream/%s/index.m3u8", name),
+			Running: false,
+			ID:      "n/a",
```

Comment:
This is an interesting question here, as now from a frontend side, you can only tell if a stream is preloaded, if they have no ID yet. I'm not sure if this is the correct approach, or if the frontend side would have any use cases for determining what is preloaded and what is not.

Right now I'm kind of fine with this indication, but I think leaving ID as an empty string would be better.

Reply:
I was mixed about that too, but also trying not to add too much to the payload. I'll change it to an empty string.